### PR TITLE
Cleanup publish test

### PR DESCRIPTION
**Pull request by toshipp**
The test contains conformance tests, the validation test that can be done by unit test, and the mount option checking. I removed the first because we used csi-test, and converted the second as a unit test. The last only remains, so I changed the test file name to be a more appropriate one.

### CODE REVIEW

**Review by cupnes — APPROVED** [review]

File: driver/node_test.go [hunk 1]
```diff
@@ -0,0 +1,16 @@
+package driver
+
+import (
+	"testing"
+
+	"github.com/container-storage-interface/spec/lib/go/csi"
+)
+
+func TestMakeMountOptions(t *testing.T) {
+	_, err := makeMountOptions(true, &csi.VolumeCapability_MountVolume{
```
Comment (llamerada-jp) [inline driver/node_test.go:10]: Could you please add one more test for normal pattern.

Comment (toshipp) [reply driver/node_test.go:10]: Do you mean a pattern for specifying read-only false and rw option? I think it is trivial case and if the pattern fails, all e2e will not pass, so there is not worth adding it.

Comment (toshipp) [reply driver/node_test.go:10]: ah, the pattern I mentioned is not normal, I doubt the normal pattern is read-only false and no options. If it is your case, I will add it.

Comment (llamerada-jp) [reply driver/node_test.go:10]: What I wanted to say is that "rw" is not specified. But, as you pointed out, all e2e will not pass. I withdraw my point.

**Review by llamerada-jp — APPROVED** [review]

**Review by llamerada-jp — APPROVED** [review]

**Pull request merged**